tft.py

```diff
@@ -1,168 +1,285 @@
+import time
+from printy import printy
+from printy import inputy
+import urllib3
+
+urllib3.disable_warnings()
+# If u havent installed your game in default path(windows) set your path here
+PATH = "default"
+auto.FAILSAFE = False
+
+class xtra:
+    def base64encode(text):
+        text = base64.b64encode(text.encode("ascii")).decode("ascii")
+        return text
+
+class lcu:
+    def connect(lockfile_path="default"):
+        if lockfile_path == "default":
+            lockfile_path = "C:\\Riot Games\\League of Legends\\lockfile"
+        if lcu.check_exist(lockfile_path):
+            return lcu.readFile(lockfile_path)
+        else:
+            raise Exception("Couldn't read lockfile!\nThis could mean that either the \
+path is not the right or the League Client is not opened!")
+
+    def check_exist(lockfile_path):
+        return os.path.exists(lockfile_path)
+
+    def readFile(lockfile_path):
+        lockfile = open(lockfile_path, "r")
+        data = lockfile.read().split(":")
+        data_dict = {
+            "port": data[2],
+            "url": "https://127.0.0.1:{}".format(data[2]),
+            "auth": "riot:{}".format(data[3]),
+            "connection_method": data[4]
+        }
+        return data_dict
+
+    def start_search(lcu_data):
+        auth = xtra.base64encode(lcu_data["auth"])
+        headers = {
+            "Content-Type": "application/json",
+            "Accept": "application/json",
+            "Authorization": F"Basic {auth}"
+        }
+        url = lcu_data["url"] + "/lol-lobby/v2/lobby/matchmaking/search"
+        request = requests.post(url, headers=headers, verify=False)
+        return request.status_code
+
+    def play_again(lcu_data):
+        auth = xtra.base64encode(lcu_data["auth"])
+        headers = {
+            "Content-Type": "application/json",
+            "Accept": "application/json",
+            "Authorization": F"Basic {auth}"
+        }
+        url = lcu_data["url"] + "/lol-lobby/v2/play-again"
+        request = requests.post(url, headers=headers, verify=False)
+        return request.status_code
+
+    def create_game_lobby_tft(lcu_data, gm:str="Normal"):
+        if gm.upper() != "NORMAL" and gm.upper() != "RANKED":
+            raise ValueError("Defined gamemode is unvalid!\nValid gamemode are Normal and Ranked.")
+        else:
+            if gm.upper() == "NORMAL":
+                q_id = 1090
+            if gm.upper() == "RANKED":
+                q_id = 1100
+        auth = xtra.base64encode(lcu_data["auth"])
+        headers = {
+            "Content-Type": "application/json",
+            "Accept": "application/json",
+            "Authorization": F"Basic {auth}"
+        }
+        data = '{{"queueId":{}}}'.format(q_id)
+        url = lcu_data["url"] + "/lol-lobby/v2/lobby/"
+        request = requests.post(url, headers=headers, data=data, verify=False)
+        return request.status_code
+
+    def create_tutorial_lobby_tft(lcu_data):

```

Comment:
Just curious, what's this method for? Just in case somebody wants to bot tutorial games instead?

Reply:
Ok I will look into the league window not accepting keyboard input issue as soon as i have free time. ^^

Reply:
> Just curious, what's this method for? Just in case somebody wants to bot tutorial games instead?

For testing purposes only
So I dont have to go into a actual game to test things